app/components/countdown_alert_component.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class CountdownAlertComponent < BaseComponent
-  attr_reader :show_at_remaining, :alert_options, :countdown_options, :tag_options
+  attr_reader :show_at_remaining, :alert_options, :countdown_options,
```

Comment:
Any reason for this change?

Reply:
That change was made by the linter. I don't think it needs to be linted in that way, so I will restore the change.